tests/InstrEngineTests/InstrEngineTests/TestMethodReplacement.cs

```diff
@@ -28,44 +28,32 @@ public static void Initialize(TestContext context)
         [Timeout(TestConstants.TestTimeout)]
         public void ReplaceMethodTest()
         {
-            //0,0,3,3,6,6,a,a,e,e
+            // Note: despite the code having two locals, Roslyn will
+            // optimize one of the locals out during compilation.
```

Comment:
I believe you can turn this off, even in retail.

Reply:
I can't see how to do that. The only optimization settings I see are setting it to Debug or Release, which corresponds to /optimize- and /optimize+.

Reply:
I think you want /optimize+ and /debug+

Reply:
Fixed, the option is hidden on the CSharpCompilationOptions class as internal methods and properties :-/